src/components/backend-ai-registry-list.ts

```diff
@@ -328,6 +332,37 @@ class BackendAIRegistryList extends BackendAIPage {
     }
   }
 
+  toggleRegistry(e, hostname) {
+    console.log(e, hostname);
+    if (!e.target.checked) {
+      this._changeRegistryState(hostname, false);
+      //this._writeUserSetting('beta_feature', false);
+      //this.beta_feature_panel = false;
+    } else {
+      this._changeRegistryState(hostname, true);
+      //this._writeUserSetting('beta_feature', true);
+      //this.beta_feature_panel = true;
+    }
+  }
+
+  _changeRegistryState(hostname, state) {
+    let input: any = Object();
+    if (state === true) {
+      this.allowed_registries.push(hostname);
+      this.notification.text = _text("registry.RegistryTurnedOn");
+    } else {
+      let index = this.allowed_registries.indexOf(hostname);
+      if (index !== 1) {
+        this.allowed_registries.splice(index, 1);
+      }
+      this.notification.text = _text("registry.RegistryTurnedOff");
+    }
+    input.allowed_docker_registries = this.allowed_registries;
```

Comment:
Local variable `input` have no role here. It seems to be safe to delete it.

Reply:
It is the remnants of old implementation (did not see the light). Fixed.